frontend/src/hocs/withHostCommunication/types.ts

```diff
@@ -108,6 +108,10 @@ export type IHostToGuestMessage = {
       type: "UPDATE_HASH"
       hash: string
     }
+  | {
+      type: "SET_CUSTOM_THEME_CONFIG"
+      theme: ICustomThemeConfig
```

Comment:
Nit: Would prefer that the `theme` field here be renamed to `themeInfo` consistent with the "SET_THEME_CONFIG" message type

Reply:
I just took a second look, and the problem is that the types aren't the same. One is an `ICustomThemeConfig` and the other is an `ExportedTheme`. Wouldn't that be confusing?

Here are the two types:
```
  | {
      type: "SET_CUSTOM_THEME_CONFIG"
      themeInfo: ICustomThemeConfig
    }
```

vs

```
  | {
      type: "SET_THEME_CONFIG"
      themeInfo: ExportedTheme
    }
```

Reply:
Also related: should I add the new fields to `ExportedTheme` as well, just for uniformity?

Reply:
IMO not confusing as they are both passing theme information - elements of theme as opposed to a complete theme - despite different types, but comfortable wherever you settle. 
Also I would vote no on adding extraneous fields simply for uniformity here - though should be examined as part of the theming refactor.